### DISCUSSION TITLE
Something went wrong error with pnp search webparts - Intermittent issue

### DISCUSSION THREAD

**Discussion by GangadharMFord** [microsoft-search/pnp-modern-search#2627] [Q&A] [answered]
Hi Team,

This is little urgent and currently we are facing the below "something went wrong" error while reading properties from js file.
For our tenant, public CDN were enabled, and this might cause the issue?
 
Search Box loading issue - Intermittent.
**Something went wrong
If the problem persists, contact the site administrator and give them the information in Technical Details.
TECHNICAL DETAILS
ERROR:
Cannot read properties of undefined (reading 'scriptResources')

CALL STACK:
TypeError: Cannot read properties of undefined (reading 'scriptResources')
    at e.configure (https://res-1.cdn.office.net/files/sp-client/chunk.systemjs-component-loader_en-us_d92702b8407f39d314da.js:1:9304)
    at https://res-1.cdn.office.net/files/sp-client/chunk.systemjs-component-loader_en-us_d92702b8407f39d314da.js:1:13758
    at Array.forEach (<anonymous>)
    at e._loadComponentImpl (https://res-1.cdn.office.net/files/sp-client/chunk.systemjs-component-loader_en-us_d92702b8407f39d314da.js:1:13720)
    at e.loadComponent (https://res-1.cdn.office.net/files/sp-client/sp-pages-assembly_en-us_037878821c0a639b1a1bff69163f3274.js:70:869169)
    at e.loadComponent (https://res-1.cdn.office.net/files/sp-client/chunk.systemjs-component-loader_en-us_d92702b8407f39d314da.js:1:13478)
    at https://res-1.cdn.office.net/files/sp-client/sp-pages-assembly_en-us_037878821c0a639b1a1bff69163f3274.js:70:855179**

**Comment by hsamdani12** [comment]
We are experiencing same thing loading the PnP Search webparts - I see it working in google chrome, and then when I try using edge all the webparts fail loading. I'm getting following CDN related error - Does anyone have any idea resolving this ?


[SPLoaderError.loadComponentError]:
Failed to load component "096b96cc-8a44-41fa-9b4d-c0ab2ab2a779" (SearchBoxWebPart). Original error: Failed to load URL 'https://publiccdn.sharepointonline.com/test123.sharepoint.com/sites/addincatalog/ClientSideAssets/890affef-33e0-4d72-bd72-36399e02143b/search-parts-searchboxwebpartstrings_en-us_4ff299871f4ee540126dd8f99a7d2c08.js' for resource 'SearchBoxWebPartStrings' in component '096b96cc-8a44-41fa-9b4d-c0ab2ab2a779' (SearchBoxWebPart). The file was not found in the server.

INNERERROR:
Failed to load URL 'https://publiccdn.sharepointonline.com/test123.sharepoint.com/sites/addincatalog/ClientSideAssets/890affef-33e0-4d72-bd72-36399e02143b/search-parts-searchboxwebpartstrings_en-us_4ff299871f4ee540126dd8f99a7d2c08.js' for resource 'SearchBoxWebPartStrings' in component '096b96cc-8a44-41fa-9b4d-c0ab2ab2a779' (SearchBoxWebPart). The file was not found in the server.

**Comment by wobba** [comment]
Upgrade? Sudden issue? Did you try to remove the sppkg and deploy again? Re-add to the site? (No need to change the pages themselves)

**Comment by hsamdani12** [comment, marked as answer]
@wobba  thanks, yes I tried to upgrade and thats when it  started giving CDN issues but now it seems to be resolved.

**Reply by wobba** [reply]
That is quite common. It takes time for the CDN to be 100% updated. Sometimes clearing localStorage and sessionStorage helps as well as client cache can be involved.

**Comment by hsamdani12** [comment]
Thanks for that information, very helpful.On Dec 14, 2022, at 11:06 AM, Mikael Svenson ***@***.***> wrote:﻿
That is quite common. It takes time for the CDN to be 100% updated. Sometimes clearing localStorage and sessionStorage helps as well as client cache can be involved.

**Comment by GangadharMFord** [comment]
thank you @wobba and all other for your valuable suggestions,

We are trying with proposed options at my tenant and will share you here the results shortly.
